### ISSUE TITLE
Error compiling daemon

### ISSUE THREAD

**Issue by bet0x** [nibroc/UptimeMonitor#4]
GCC 
```
barrahome@barrahome.org [~/proyectos/UptimeMonitor/daemon]# gcc -v
Usando especificaciones internas.
Objetivo: x86_64-redhat-linux
Configurado con: ../configure --prefix=/usr --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=http://bugzilla.redhat.com/bugzilla --enable-bootstrap --enable-shared --enable-threads=posix --enable-checking=release --with-system-zlib --enable-__cxa_atexit --disable-libunwind-exceptions --enable-gnu-unique-object --enable-languages=c,c++,objc,obj-c++,java,fortran,ada --enable-java-awt=gtk --disable-dssi --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-1.5.0.0/jre --enable-libgcj-multifile --enable-java-maintainer-mode --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --disable-libjava-multilib --with-ppl --with-cloog --with-tune=generic --with-arch_32=i686 --build=x86_64-redhat-linux
Modelo de hilos: posix
gcc versión 4.4.7 20120313 (Red Hat 4.4.7-17) (GCC)
barrahome@barrahome.org [~/proyectos/UptimeMonitor/daemon]#

```
```
barrahome@barrahome.org [~/proyectos/UptimeMonitor/daemon]# make
cc -O3   -c -o uptimed.o uptimed.c
cc -O3   -c -o notifier.o notifier.c
notifier.c:16: error: redefinición de la definición de tipo ‘Notifier’
notifier.h:18: nota: la declaración previa de ‘Notifier’ estaba aquí
make: *** [notifier.o] Error 1
barrahome@barrahome.org [~/proyectos/UptimeMonitor/daemon]#
```

**Comment by nibroc** [comment]
Hello, and thanks for the interest in this project. While it did reach the point of being usable (and is what I actually use to monitor some of my own personal servers), I can't really recommend you actually use this project unless you're just looking for something to hack on. This has never been used extensively, and I can't in good faith say that I believe it to be free of defects either functionally or security related.

In short, while I'd happily welcome someone hacking on it or using it for example code or something, I fear I must caution against using this for real.

Anyway, whenever I get a chance this weekend, I'll try to fix up the compilation error.

**Comment by nibroc** [comment]
I've never actually compiled this on a version of GCC so old, but since it wasn't using any C11 features anyway, I bumped the version down to C99. I think that was actually the issue. I had the CFLAGS stuff done incorrectly in the Makefile, so your compiler was defaulting to C89, a standard in which the way I'd done the struct declaring was invalid. Anyway, things should be fixed now, at least hopefully. The oldest compiler I've ever tested it on is GCC 4.8 :).
